versioned/mongodb/src/test/java/com/dremio/nessie/versioned/impl/condition/TestMongoDbExpressions.java

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright (C) 2020 Dremio
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.dremio.nessie.versioned.impl.condition;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.Arrays;
+import java.util.Random;
+
+import org.bson.Document;
+import org.junit.jupiter.api.Test;
+
+
+import com.dremio.nessie.versioned.impl.SampleEntities;
+import com.dremio.nessie.versioned.store.Entity;
+import com.dremio.nessie.versioned.store.HasId;
+import com.dremio.nessie.versioned.store.Store;
+
+public class TestMongoDbExpressions {
+
+  private final Entity av0 = Entity.ofBoolean(true);
+  private final Entity av1 = Entity.ofBoolean(false);
+  private final Entity av2 = Entity.ofString("mystr");
+  private Random random = new Random(8612341233543L);
+
+  private final Entity id  = Entity.ofString(((HasId) SampleEntities.createL1(random)).getId().toString());
```

Comment:
L1 is a HasId, the cast seems rendundant.

Reply:
IntelliJ complained earlier, but not it seems to be ok. Removed.